js/.eslintrc.js

```diff
@@ -3,3 +3,6 @@
 // http://www.apache.org/licenses/LICENSE-2.0
 
 module.exports = require('@attic/eslintrc');
+
+// TODO: Include this in @attic/eslintrc.
+module.exports.rules['eqeqeq'] = ['error', 'allow-null'];
```

Comment:
That's what I thought. Shame.

Right below they also say "use shortcuts in `if` statements, e.g. instead of `if (foo === '') use `if (foo)`", which is dangerous advice in this context.